cuda/test/solver/bicgstab_kernels.cpp

```diff
@@ -69,7 +69,7 @@ class Bicgstab : public ::testing::Test {
         cuda = gko::CudaExecutor::create(0, ref);
 
         mtx = gen_mtx(123, 123);
-        make_diag_dominant(mtx.get());
+        gko::test::make_diag_dominant(mtx.get());
```

Comment:
We should remove this. BiCGSTAB should be able to solve general systems. Or atleast add another test with a non-diagonally dominant system.

Reply:
When I removing it, it gives different result...
The matrix might be singular when it is not (strongly) diag dominant, so it has several solution in this sense?